### DISCUSSION TITLE
Crazyflies loosing pose when using Crazyswarm.

### DISCUSSION THREAD

**Discussion by WoodyloveBo** [bitcraze/discussions#1146] [Q&A]
hello.

I would like to mention an issue I am having while using the Crazyswarm API (ROS1 version). I'm using Crazyswarm, but I'm having issues with Crazyflie not flying as instructed and losing its route along the way.

Here are my settings:

Four drones communicate with two Crazyradio PAs via two different channels (80,100).

There are a total of 4 Motion Capture Cameras using Optitrack.

https://github.com/bitcraze/discussions/assets/143582301/c4b6db9f-e3cf-43c6-bb31-c0438b9e3615

Currently, the frequency at which Motion Capture data is received is 100, which is fine. Considering wireless signal interference, I experimented with changing the channel to 60, 70, 80, 90, and 100, but the same problem occurs. I would appreciate it if you could let me know what the problem is with this.

thank you

**Comment by knmcguire** [comment]
Hi! This is a double post of the discussion on crazyswarm1: https://github.com/USC-ACTLab/crazyswarm/discussions/784. That is the proper location for this support, but do mind that all the efforts have pretty much gone to [Crazyswarm2](https://github.com/IMRCLab/crazyswarm2)
